src/main/scala/it/pagopa/interop/catalogprocess/common/readmodel/ReadModelCatalogQueries.scala

```diff
@@ -173,12 +173,15 @@ object ReadModelCatalogQueries extends ReadModelQuery {
       case Seq()      => None
       case attributes =>
         Some(Filters.or {
-          Filters.in("data.descriptors.attributes.certified.id.id", attributes.map(_.toString))
-          Filters.in("data.descriptors.attributes.certified.id.ids", attributes.map(_.toString))
-          Filters.in("data.descriptors.attributes.declared.id.id", attributes.map(_.toString))
-          Filters.in("data.descriptors.attributes.declared.id.ids", attributes.map(_.toString))
-          Filters.in("data.descriptors.attributes.verified.id.id", attributes.map(_.toString))
-          Filters.in("data.descriptors.attributes.verified.id.ids", attributes.map(_.toString))
+          Filters.elemMatch("data.descriptors.attributes.certified.0", Filters.in("id", attributes.map(_.toString)))
+          Filters.elemMatch("data.descriptors.attributes.certified.1", Filters.in("id", attributes.map(_.toString)))
+
+          Filters.elemMatch("data.descriptors.attributes.declared.0", Filters.in("id", attributes.map(_.toString)))
+          Filters.elemMatch("data.descriptors.attributes.declared.1", Filters.in("id", attributes.map(_.toString)))
+
+          Filters.elemMatch("data.descriptors.attributes.verified.0", Filters.in("id", attributes.map(_.toString)))
+          Filters.elemMatch("data.descriptors.attributes.verified.1", Filters.in("id", attributes.map(_.toString)))
+
```

Comment:
The new structure is:
```
{
   "data": {
      "descriptors": [
         {
            "attributes": {
               "certified": [["a", "b"], ["c"]],
               "declared": [["d", "e", "f"], ["g", "h"], ["i"]],
               "verified": []
            }
         }
      ]
   }
}
```
The implementation proposed does not work. I just tested it directly on DocumentDB.

Reply:
attributes: {
            certified: [
              [
                {
                  explicitAttributeVerification: true,
                  id: '2edd6cea-39cc-4cd4-b83a-f650d4e130b7'
                }
              ]
            ],
            declared: [],
            verified: [
              [
                {
                  explicitAttributeVerification: true,
                  id: '1c6b7e96-90e3-4309-8c98-7faf65743780'
                }
              ]
            ]

Reply:
Ok I just missed the inside object (below you can find the right model without explicitAttributeVerification), but this doesn't change what I wrote: that solution doesn't work.
{
   "data": {
      "descriptors": [
         {
            "attributes": {
               "certified": [[{"id": "a"}, {"id": "b"}], [{"id": "c"}]],
               "declared": [[{"id": "d"}, {"id": "e"}, {"id": "f"}], [{"id": "g"}, {"id": "h"}], [{"id": "i"}]],
               "verified": []
            }
         }
      ]
   }
}
Did you try to execute it directly on DocumentDB?

Reply:
It works!!!

db.getCollection('eservices').find( { $or: [
{'data.descriptors.attributes.certfied.0.id': { $in: ["1c6b7e96-90e3-4309-8c98-7faf65743780"]}},
{'data.descriptors.attributes.certfied.1.id': { $in: ["1c6b7e96-90e3-4309-8c98-7faf65743780"]}},
{'data.descriptors.attributes.declared.0.id': { $in: ["1c6b7e96-90e3-4309-8c98-7faf65743780"]}},
{'data.descriptors.attributes.declared.1.id': { $in: ["1c6b7e96-90e3-4309-8c98-7faf65743780"]}},
{'data.descriptors.attributes.verified.0.id': { $in: ["1c6b7e96-90e3-4309-8c98-7faf65743780"]}},
{'data.descriptors.attributes.verified.1.id': { $in: ["1c6b7e96-90e3-4309-8c98-7faf65743780"]}}
]})

Reply:
I think this is the right solution:
`db.eservices.find({ "data.descriptors" : {$elemMatch: { "attributes.certified" : { $elemMatch : {$elemMatch : {"id" : { $in : ["<ID_ATTIBUTE>"]}}}}}}})`

Reply:
I tested also with array with length greater than 2, it works

Reply:
This is your solution:
```
db.getCollection('eservices').find( { "data.id" : "47573bdb-bdd2-4ff2-a6e6-d0f1dc2c83cc",  $or: [
  {'data.descriptors.attributes.certfied.0.id': { $in: ["024c8995-91d4-49f1-937d-e83fd4ee8803"]}},
  {'data.descriptors.attributes.certfied.1.id': { $in: ["024c8995-91d4-49f1-937d-e83fd4ee8803"]}},
  {'data.descriptors.attributes.declared.0.id': { $in: []}},
  {'data.descriptors.attributes.declared.1.id': { $in: []}},
  {'data.descriptors.attributes.verified.0.id': { $in: []}},
  {'data.descriptors.attributes.verified.1.id': { $in: []}}
  ]})
```
no recordswere  retrieved.
This is my suggestion:
```
db.getCollection('eservices').find({ "data.id": "47573bdb-bdd2-4ff2-a6e6-d0f1dc2c83cc", "data.descriptors.attributes.certified" : { $elemMatch : {$elemMatch : {"id" : { $in : ["024c8995-91d4-49f1-937d-e83fd4ee8803"]}}}}})
```
it returns the record correctly.

Reply:
Got it!

db.getCollection('eservices').find( { "data.descriptors.attributes.verified": { $elemMatch: { $elemMatch: { "id": { $in: ["efd95d03-de47-4347-9b60-5072fcb23e2c"] } } } } } )

Reply:
I was wrong